bugsnag-android-core/src/test/java/com/bugsnag/android/BreadcrumbMutabilityTest.kt

```diff
@@ -1,24 +1,12 @@
 package com.bugsnag.android
 
 import org.junit.Assert.assertFalse
-import org.junit.Assert.assertNotSame
-import org.junit.Assert.assertTrue
 import org.junit.Test
 
 class BreadcrumbMutabilityTest {
-
-    @Test
-    fun breadcrumbCopiesMap() {
-        val data = mutableMapOf<String, Any>()
-        val breadcrumb = Breadcrumb("foo", BreadcrumbType.MANUAL, data)
-        data["a"] = "bar"
-        assertTrue(breadcrumb.metadata.isEmpty())
-        assertNotSame(data, breadcrumb.metadata)
-    }
```

Comment:
Has not making a copy been specified as the behaviour?

Reply:
Yes, the map is now accessible and mutable, so we no longer need to assert that these objects are not the same. Mutating the map is already covered by other tests.